full-stack-asset-transfer-guide/docs/ApplicationDev/01-FabricGateway.md

```diff
@@ -48,7 +48,7 @@ In order for the client application to operate effectively, it must make use of
 
 ## Fabric Gateway client API
 
-The following diagram demonstrates for the transaction subsmit flow is executed for a client using the Fabric Gateway client API. Solid orange lines represent interactions between the client and a Gateway peer, which must cross the firewall at the boundary of the network deployment. Dashed green lines represent interactions between network nodes.
+The following diagram demonstrates for the transaction submit flow how it is executed for a client using the Fabric Gateway client API. Solid orange lines represent interactions between the client and a Gateway peer, which must cross the firewall at the boundary of the network deployment. Dashed green lines represent interactions between network nodes.
```

Comment:
I think I would word this as, "The following diagram demonstrates how the transaction submit flow is executed for a client using the Fabric Gateway client API." This matches the section on legacy SDK behaviour and I think is what I originally intended before the typo on the word "how".

Reply:
Maybe change it to:
`The following diagram demonstrates **how** the transaction submit flow is executed for a client using the Fabric Gateway client API. Solid orange lines represent interactions between the client and a Gateway peer, which must cross the firewall at the boundary of the network deployment. Dashed green lines represent interactions between network nodes.`
change the for to how? I saw the typo in submit and corrected it but still didn't find it easy to read ... maybe changing the first for to how would make it easier?

Reply:
oh! Now I see what you meant by Legacy, there I corrected it.